mlflow/langchain/__init__.py

```diff
@@ -297,10 +351,67 @@ def log_model(
 
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
+    :param loader_fn: A function that takes a string `persist_dir` defined below as the argument
+                      and returns a unserializable object. This is required for models containing
+                      unserializable objects. For example, RetrievalQA models require a retriever
+                      (https://python.langchain.com/docs/modules/data_connection/retrievers/),
+                      VectorDBQA and VectorDBQAWithSourcesChain models require a vectorstore
+                      (https://python.langchain.com/docs/modules/data_connection/vectorstores/),
+                      APIChain models require a requests_wrapper(https://python.langchain.com/docs
+                      /modules/agents/tools/integrations/requests), HypotheticalDocumentEmbedder
+                      require an embeddings (https://python.langchain.com/docs/modules/data_connec
+                      tion/text_embedding/) and SQLDatabaseChain require a database (https://pytho
+                      n.langchain.com/docs/modules/agents/toolkits/sql_database).
```

Comment:
/ai This is docstring in Python. Please rewrite the links using Link syntax in reStructured